src/charactersheet/viewmodels/common/wizard/index.js

```diff
@@ -142,6 +148,7 @@ export class WizardViewModel {
                 profileImage: { type: 'email' },
                 health: { maxHitPoints: 10 },
                 traits: traits,
+                features: features,
```

Comment:
does this require an API change or it will persist without issue?

Reply:
It does yes. There should be a Park for it